cluster/gce/gci/configure-helper.sh

```diff
@@ -656,7 +656,16 @@ contexts:
 EOF
   fi
 
-if [[ -n "${GCP_IMAGE_VERIFICATION_URL:-}" ]]; then
+  if [[ -n "${GENERIC_ADMISSION_KUBECONFIG:-}" ]]; then
+    # This is the config file for generic admission webhooks
+    cat <<EOF >/etc/generic_admission.config
+${GENERIC_ADMISSION_KUBECONFIG}
+EOF
+    GENERIC_ADMISSION_CONFIG="kubeConfigFile: /etc/gcp_generic_admission.config
```

Comment:
Minor nit, maybe it'd be more readable if you leave out the newline here and instead split line 699 in half?